oggm/core/models/flowline.py

```diff
@@ -1653,3 +1653,63 @@ def iterative_initial_glacier_search(gdir, y0=None, init_bias=0., rtol=0.005,
         _ = past_model.run_until_and_store(y1, path=path)
     else:
         past_model.to_netcdf(path)
+
+@entity_task(log)
+def run_from_gcm(gdir, filename='cesm_data', filesuffix='', **kwargs):
+    """ Runs glacier with climate input from a general circulation model.
+
+     Parameters
+     ----------
+     y0 : is the start year of the model run
+     y1 : is the end year of the model run
+     filename: name of the climate file. (Can be made with the
+     process_cesm_data function)
+     filesuffix : for the input & output file
+     kwargs : kwargs to pass to the FluxBasedModel instance
+     """
+
+    if cfg.PARAMS['use_optimized_inversion_params']:
+        d = gdir.read_pickle('inversion_params')
+        fs = d['fs']
+        glen_a = d['glen_a']
+    else:
+        fs = cfg.PARAMS['flowline_fs']
+        glen_a = cfg.PARAMS['flowline_glen_a']
+
+    kwargs.setdefault('fs', fs)
+    kwargs.setdefault('glen_a', glen_a)
+
+    y0 = cfg.PARAMS['y0']
+    ye = cfg.PARAMS['y1']
```

Comment:
aha I see. Can you also change the name of the params to ``ys``, ``ye`` in the ``params.cfg`` file?

You can keep the idea of having them as keyword arguments in the function and use the params as default if the user didn't set them.